src/cli/dto/gen-dtos.ts

```diff
@@ -27,11 +33,88 @@ export class GenDTOs {
     return fileContent
   }
 
+  private static removeCircularReferences(obj: any): CircularRefInfo[] {
```

Comment:
can we add tests for this?

Reply:
https://github.com/Kong/brij/pull/2/commits/d42c737ce193d52393c09e1b9068fc261773b546